sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapExamples.cs

```diff
@@ -175,5 +176,90 @@ public void CreditDefaultSwapGetPortfolioCashFlowsExample(ModelSelection.ModelEn
+                mktContext,
+                pricingContext,
+                description: "Isda cds valuation demo"
+            );
+
+            // UPSERT the configuration recipe
+            var upsertRecipeRequest = new UpsertRecipeRequest(recipe, null);
+            var upsertRecipeResponse = _recipeApi.UpsertConfigurationRecipe(upsertRecipeRequest);
+            Assert.That(upsertRecipeResponse.Value, Is.Not.Null);
+
+            // CREATE the required market data, in this case a credit curve and a yield curve
+            string ycXml = File.ReadAllText("../../../tutorials/Ibor/ExampleMarketData/IsdaYieldCurve_USD_20200605.xml");
+            var ycOpaque = new OpaqueMarketData(ycXml, "xml", "Example isda yield curve", ComplexMarketData.MarketDataTypeEnum.OpaqueMarketData);
+            var upsertYcId = new ComplexMarketDataId("Lusid", effectiveAt: testNow, marketAsset: "IsdaYieldCurve/USD");
+            var ccData = new CreditSpreadCurveData(
+                baseDate: testNow,
+                domCcy: "USD",
+                tenors: new List<string> {"6M", "1Y", "5Y", "10Y"},
+                spreads: new List<decimal> {0.001m, 0.0011m, 0.0015m, 0.002m},
+                recoveryRate: 0.4m,
+                marketDataType: ComplexMarketData.MarketDataTypeEnum.CreditSpreadCurveData);
+            var upsertCcId = new ComplexMarketDataId("Lusid", effectiveAt: testNow, marketAsset: "IsdaCreditCurve/XYZCorp/USD");
+
+            // UPSERT market data
+            var upsertYcRequest = new UpsertComplexMarketDataRequest(upsertYcId, ycOpaque);
+            var upsertCcRequest = new UpsertComplexMarketDataRequest(upsertCcId, ccData);
+            var upsertCmdResponse = _complexMarketDataApi.UpsertComplexMarketData(testScope,
+                new Dictionary<string, UpsertComplexMarketDataRequest>
+                {
+                    {"yc", upsertYcRequest},
+                    {"cc", upsertCcRequest}
+                });
+            Assert.That(upsertCmdResponse.Failed, Is.Empty);
+
+            // CREATE a inline valuation request with an example CDS instrument
+            var pvKey = "Holding/default/PV";
+            var valRequest = new InlineValuationRequest(
+                recipeId: new ResourceId(testScope, recipeName),
+                metrics: new List<AggregateSpec>
+                {
+                    new AggregateSpec("Analytic/default/InstrumentTag", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec("Analytic/default/ValuationDate", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec(pvKey, AggregateSpec.OpEnum.Value),
+                },
+                reportCurrency: "USD",
+                valuationSchedule: new ValuationSchedule(effectiveAt: new DateTimeOrCutLabel(testNow)),
+                instruments: new List<WeightedInstrument>
+                {
+                    new WeightedInstrument(1, $"{cds.Ticker}-{cds.FlowConventions.Currency}-TestCds", cds)
```

Comment:
state that this is a correlation id. It should be unique per call but otherwise serves no purpose other than for the client/user to identify the item(s) in the result.

Reply:
this is inline, and defines the weighting of the instrument, it's not a correlation id (although I explained the correlation in the market data upsert)